### ISSUE TITLE
Controller stuck in middle of screen

### ISSUE THREAD

**Issue by JustinTychek** [codebicycle/videospeed#213]
This is specifically on Hulu in Firefox. The controller will stay in the middle of the screen and if I move it to one of the corners, it will default back to the middle of the screen whenever I pause the video or move to another window. This used to also be an issue on Chrome for me but it got resolved years ago so I think it may have to do with Firefox being on an older version of the extension than Chrome. @codebicycle doesn't seem to be looking at the feedback section so who knows if this will even be seen

**Comment by markygnlg** [comment]
Try pressing  "V"

![My Video Speed zController Serttings](https://github.com/user-attachments/assets/4d379ff3-5ce3-40cb-9ea1-dfc477cb4195)

**Comment by JustinTychek** [comment]
Yes, I know that I can press V to hide the player. That's what I've been using as a solution but not what the core problem is. On all other sites, the playback speed is shown in the top left corner and I like to have it visible there so I can know what speed I'm on at a glance. On Hulu, it is stuck in the middle of the screen so I can't keep the speed visible or it will be a major distraction. Again, this isn't an issue on Chrome, only Firefox from what I've seen.

**Comment by JustinTychek** [comment]
reopening, read above comment
